subprojects/file-watching/src/test/groovy/org/gradle/internal/watch/vfs/impl/WatchingVirtualFileSystemTest.groovy

```diff
@@ -47,15 +47,15 @@ class WatchingVirtualFileSystemTest extends Specification {
     def "invalidates the virtual file system before and after the build when watching is disabled"() {
         when:
         rootReference.update { root -> nonEmptySnapshotHierarchy }
-        watchingVirtualFileSystem.afterBuildStarted(false, false, buildOperationRunner)
+        watchingVirtualFileSystem.afterBuildStarted(false, false, false,buildOperationRunner)
```

Comment:
```suggestion
        watchingVirtualFileSystem.afterBuildStarted(false, false, false, buildOperationRunner)
```

Nit. That also happens for the places where the new flag has been added.

Reply:
Damn IntelliJ inline labels...